src/Cursor.php

```diff
@@ -477,4 +502,14 @@ public function getColumn(): int
     {
         return $this->column;
     }
+
+    public function getEncoding(): string
```

Comment:
Surely adding return types is the opposite of a performance optimization. This will make it slower, at runtime.